<launch>
  <node name="control_offline_r1" pkg="franka_coord" type="control_offline" output="screen">
    <param name="robot_id" value="1" /> 
  </node>
  <node name="control_offline_r2" pkg="franka_coord" type="control_offline" output="screen">
    <param name="robot_id" value="2" /> 
  </node>

  <node name="control_offline_r3" pkg="franka_coord" type="control_offline" output="screen">
    <param name="robot_id" value="3" /> 
  </node>

  <!-- <node name="control_offline_r4" pkg="franka_coord" type="control_offline" output="screen">
    <param name="robot_id" value="4" /> 
  </node>

  <node name="control_offline_r5" pkg="franka_coord" type="control_offline" output="screen">
    <param name="robot_id" value="5" /> 
  </node>

  <node name="control_offline_r6" pkg="franka_coord" type="control_offline" output="screen">
    <param name="robot_id" value="6" /> 
  </node>

  <node name="control_offline_r7" pkg="franka_coord" type="control_offline" output="screen">
    <param name="robot_id" value="7" /> 
  </node>

 <node name="control_offline_r8" pkg="franka_coord" type="control_offline" output="screen">
    <param name="robot_id" value="8" /> 
  </node>

  <node name="control_offline_r9" pkg="franka_coord" type="control_offline" output="screen">
    <param name="robot_id" value="9" /> 
  </node>

  <node name="control_offline_r10" pkg="franka_coord" type="control_offline" output="screen">
    <param name="robot_id" value="10" /> 
  </node> -->
  
</launch>